src/applications/check-in/components/pages/Edit/shared/Header.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { EDITING_PAGE_NAMES } from '../../../../utils/appConstants';
+
+export default function Header(props) {
+  const { what, editingPage, value } = props;
+  const addOrEdit = value ? 'Edit' : 'Add';
+  const getWhoseFromPage = () => {
+    if (editingPage === EDITING_PAGE_NAMES.DEMOGRAPHICS) {
+      return 'your';
+    }
+    if (editingPage === EDITING_PAGE_NAMES.EMERGENCY_CONTACT) {
+      return "emergency contact's";
```

Comment:
tiny nitpick but it looks like for emergency contact the designs are just going with `your contact`

Reply:
@brianseek Yes, I had a slack thread with UX about it, we settled on the full thing https://dsva.slack.com/archives/C02GXKL8WM6/p1647015309285989